examples/SimplePerTitleEncodingWithMP4.php

```diff
@@ -0,0 +1,425 @@
+
+    //CREATE S3 INPUT
+    $input = new S3Input($inputS3Bucketname, $inputS3AccessKey, $inputS3SecretKey);
+    $input->setName("AWS S3 Input Bucket");
+    $input = $apiClient->inputs()->s3()->create($input);
+
+    //CREATE S3 OUTPUT
+    $output = new S3Output($outputS3Bucketname, $outputS3AccessKey, $outputS3SecretKey);
+    $output->setName("AWS S3 Output Bucket");
+    $output = $apiClient->outputs()->s3()->create($output);
+
+    //CREATE CRF ENCODINGS
+    $activeCrfEncodings = array();
+    foreach ($videoFiles as $key => $videoFile)
+    {
+        $inputPath = $videoFile['inputPath'];
+        $outputPath = $videoFile['outputPath'];
+        $encodingName = "CRF | " . $videoFile['encodingName'];
+        //RUN CRF ENCODING
+        $videoFiles[$key]['crfEncoding'] = runFastCrfEncoding($apiClient, $encodingName, $input, $inputPath, $output, $outputPath);
+    }
+
+    //WAIT UNTIL ALL CRF ENCODINGS ARE FINISHED
+    $allCrfFinished = false;
+    do
+    {
+        $states = array();
+        foreach ($videoFiles as $key => $videoFile)
+        {
+            /** @var Encoding $currentCrfEncoding */
+            $currentCrfEncoding = $videoFile['crfEncoding'];
+            $status = $apiClient->encodings()->status($currentCrfEncoding);
+            $isRunning = !in_array($status->getStatus(), array(Status::ERROR, Status::FINISHED));
+            $states[] = $isRunning;
+            $currentTimestamp = date_create(null, new DateTimeZone('UTC'))->getTimestamp();
+            echo $currentTimestamp . ": " . $currentCrfEncoding->getName() . " => " . $status->getStatus() . "\n";
+        }
+        $allCrfFinished = !in_array(true, $states);
+        sleep(ENCODING_STATUS_REFRESH_RATE);
+    } while (!$allCrfFinished);
+
+    //START PER TITLE ENCODINGS
+    $staticVideoEncodingConfigs = array();
+    $perTitleVideoEncodingConfigs = array();
+    $activeEncodings = array();
+    foreach ($videoFiles as $videoFile)
+    {
+        /** @var Encoding $currentCrfEncoding */
+        $currentCrfEncoding = $videoFile['crfEncoding'];
+        $inputPath = $videoFile['inputPath'];
+        $outputPath = $videoFile['outputPath'];
+        $encodingName = "Per Title | " . $videoFile['encodingName'];
+        $crfMuxing = $apiClient->encodings()->muxings($videoFile['crfEncoding'])->fmp4Muxing()->listPage()[0];
+
+        // CREATE ENCODING
+        $encoding = new Encoding($encodingName);
+        $encoding->setCloudRegion(CloudRegion::GOOGLE_EUROPE_WEST_1);
+        $encoding = $apiClient->encodings()->create($encoding);
+
+        //CREATE VIDEO/AUDIO INPUT STREAMS
+        $inputStreamVideo = new InputStream($input, $inputPath, SelectionMode::AUTO);
+
+        // CREATE VIDEO CODEC CONFIGURATIONS
+        foreach ($bitrateLadderEntries as $bitrateLadderEntry)
+        {
+            if ($bitrateLadderEntry["codec"] !== "h264")
+                continue;
+
+            $mp4MuxingOutputPath = $outputPath . 'mp4-per-title/';
```

Comment:
do not append an additional output path that might not be intended by a customer